content_schemas/formats/popular_links.jsonnet

```diff
@@ -0,0 +1,40 @@
+{
+  document_type: "popular_links",
+  base_path: "optional",
+  routes: "optional",
+  rendering_app: "optional",
+  title: "optional",
+  redirects: "optional",
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        links_details: {
+          type: "array",
+          minItems: 6,
+          maxItems: 6,
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "title",
+              "url"
+            ],
+            properties: {
+              title: {
+                type: "string"
+              },
+              url: {
+                type: "string"
+              }
+            }
+          }
+        },
+        language: {
```

Comment:
Is language the same thing as `locale`?

If so, we shouldn't need it inside details - the usual way would be to have different `documents` for the different `locale`s.